django/contrib/auth/hashers.py

```diff
@@ -190,8 +190,8 @@ def _load_library(self):
 
     def salt(self):
         """Generate a cryptographically secure nonce salt in ASCII."""
-        # 12 returns a 71-bit value, log_2((26+26+10)^12) =~ 71 bits
-        return get_random_string(12)
+        # 12 returns a 71-bit value, log_2(len(RANDOM_STRING_CHARS)^12) =~ 71 bits
+        return get_random_string(12, RANDOM_STRING_CHARS)
```

Comment:
It's not so obvious why `RANDOM_STRING_CHARS` (the default value) is passed (for clarity with the comment, I guess?)

Also, it seems more readable to pass them as kwargs rather than as args.

Reply:
> It's not so obvious why `RANDOM_STRING_CHARS` (the default value) is passed (for clarity with the comment, I guess?)

Yes and because we will use it in few more places, see #12553.

> Also, it seems more readable to pass them as kwargs rather than as args.

I'm going to work on #12553 tomorrow, so I can change it to a keyword argument there.